minigotchi-ESP32/minigotchi-ESP32.ino

```diff
@@ -28,7 +28,7 @@ void loop() {
     // the longer we are on this channel, the more likely we're gonna see a pwnagotchi on this channel
     // get local payload from local pwnagotchi, send raw frame if one is found
     minigotchi.detect();
-    delay(250);
+    delay(900);
```

Comment:
probably if we're going to up the delay, it needs to remain somewhat consistent with a pwnagotchi's speed and the minigotchi's upstream code...

Reply:
Resolved.